.eslintignore

```diff
@@ -0,0 +1 @@
+/**/*.d.ts
```

Comment:
Added this because of an eslint conflic, which mistakenly treats .d.ts as js and gets confused